packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -71,164 +77,250 @@
   void onError(String errorCode, String errorMessage);
 }
 
-public class Camera {
+/**
+ * Note: at this time we do not implement zero shutter lag (ZSL) capture. This is a potential
```

Comment:
The primary documentation for the class should explain what the class is/does at a high level, not consist solely of an implementation detail note.

This comment is useful, but this isn't the right place for it; I would actually suggest removing it from the code entirely and filing it as a feature request, with all this as supporting info.

Reply:
@acoutts I don't really have a good view on this. Could you maybe look into this please?

Reply:
This was just a shower thought I had, I agree it's fine to just remove it.